HOW-TO-START-DEVELOPMENT.md

```diff
@@ -0,0 +1,45 @@
+# How to start contributing
+
+## Read kubebuilder documentation
+[Docs](https://book.kubebuilder.io/introduction)
+
+## Build your develop environment
+
+### Easy way
+1. Download and install [kind](https://kind.sigs.k8s.io/docs/user/quick-start/#creating-a-cluster).
+2. Create lind cluster `kind create cluster --name etcd-operator-kind`.
+3. Switch kubectl context to kind `kubectl cluster-info --context kind-etcd-operator-kind`. Be attentive to avoid damaging your production environment.
```

Comment:
I would write it like `kubectl set-context kind-etcd-operator-kind` or something like that. The command you provided does not change context. Following `make` commands will be executed against some other cluster :)

Reply:
Thank you, will have a look today.
Probably I accidentally copied the wrong line from my history. At least I executed everything against correct cluster :)

Reply:
@sircthulhu fixed